examples/with-backstage/backstage-humanitec.tf

```diff
@@ -0,0 +1,118 @@
+resource "humanitec_application" "backstage" {
+  id   = "backstage"
+  name = "backstage"
+}
+
+# Configure required values for backstage
+
+resource "humanitec_value" "backstage_github_org_id" {
+  app_id      = humanitec_application.backstage.id
+  key         = "GITHUB_ORG_ID"
+  description = ""
+  value       = var.github_org_id
+  is_secret   = false
+}
+
+resource "humanitec_value" "backstage_github_app_id" {
+  app_id      = humanitec_application.backstage.id
+  key         = "GITHUB_APP_ID"
+  description = ""
+  value       = local.github_app_id
+  is_secret   = false
+}
+
+resource "humanitec_value" "backstage_github_app_client_id" {
+  app_id      = humanitec_application.backstage.id
+  key         = "GITHUB_APP_CLIENT_ID"
+  description = ""
+  value       = local.github_app_client_id
+  is_secret   = true
+}
+
+resource "humanitec_value" "backstage_github_app_client_secret" {
+  app_id      = humanitec_application.backstage.id
+  key         = "GITHUB_APP_CLIENT_SECRET"
+  description = ""
+  value       = local.github_app_client_secret
+  is_secret   = true
+}
+
+resource "humanitec_value" "backstage_github_app_private_key" {
+  app_id      = humanitec_application.backstage.id
+  key         = "GITHUB_APP_PRIVATE_KEY"
+  description = ""
+  value       = indent(2, local.github_app_private_key)
+  is_secret   = true
+}
+
+resource "humanitec_value" "backstage_github_app_webhook_secret" {
+  app_id      = humanitec_application.backstage.id
+  key         = "GITHUB_APP_WEBHOOK_SECRET"
+  description = ""
+  value       = local.github_webhook_secret
+  is_secret   = true
+}
+
+resource "humanitec_value" "backstage_humanitec_org" {
+  app_id      = humanitec_application.backstage.id
+  key         = "HUMANITEC_ORG_ID"
+  description = ""
+  value       = var.humanitec_org_id
+  is_secret   = false
+}
+
+resource "humanitec_value" "backstage_humanitec_token" {
+  app_id      = humanitec_application.backstage.id
+  key         = "HUMANITEC_TOKEN"
+  description = ""
+  value       = var.humanitec_ci_service_user_token
+  is_secret   = true
+}
+
+resource "humanitec_value" "backstage_cloud_provider" {
+  app_id      = humanitec_application.backstage.id
+  key         = "CLOUD_PROVIDER"
+  description = ""
+  value       = local.cloud_provider
+  is_secret   = false
+}
+
+# Configure required resources for backstage
+
+locals {
+  res_def_prefix = "backstage-"
+}
+
+# in-cluster postgres
+
+module "backstage_postgres" {
+  # Not pinned as we don't have a release yet
+  # tflint-ignore: terraform_module_pinned_source
+  source = "git::https://github.com/humanitec-architecture/resource-packs-in-cluster.git//humanitec-resource-defs/postgres/basic?ref=main"
```

Comment:
Why are you using in cluster dbs instead of Azure ones?

Reply:
Mainly to reduce costs and because it is much quicker to create the ref-arch like this, we follow a similar approach in AWS/GCP.